Update README (#35)
slack-samples/deno-starter-template: README.md

$ cd my-app
```

## Running Your Project Locally

While building your app, you can see your changes propagated to your workspace
in real-time with `slack run`. You'll know an app is the development version if
the name has the string `(local)` appended.

```zsh
# Run app locally
$ slack run

Connected, awaiting events
```

To stop running locally, press `<CTRL> + C` to end the process.

## Creating Triggers

> When you run or deploy your project, the CLI will prompt you to create a new

Reviewer: Maybe to make it slightly clearer / push to action, we monospace the two CLI action words?

```suggestion
When you `run` or `deploy` your project, the CLI will prompt you to create a new
```

> trigger if one is found in the `triggers/` directory.

Reviewer: I think this behaviour also only happens if no trigger exists for the particular app instance (app+workspace).

Worth calling that out here or no?

Author: Oh, really? So, if someone evolves the app and later adds a trigger, it won't get picked up?

Author: Did a bit of editing and rearranging. Let me know what you think, but the nod added is:

> When you `run` or `deploy` your project for the first time, the CLI will prompt
you to create a trigger if one is found in the `triggers/` directory. For any
subsequent triggers added to the application, each must be
[manually added using the `trigger create` command](#manual-trigger-creation).

Reviewer: Perfect, thank you!

When creating a trigger, you must select the workspace and environment that

> you'd like to create the trigger in. Each workspace has a local development

Reviewer 2: ```suggestion
you'd like to create the trigger in. Each workspace can have a local development
```

$ slack deploy
```

> After deploying, you'll be prompted to

Reviewer: ```suggestion
After deploying for the first time, you'll be prompted to
```

[create a new link trigger](#creating-triggers) for the production version of
your app. When that trigger is invoked, the workflow should run just as it did

> in when developing locally (but without requiring your server to be running).

Reviewer: Just me or is the wording in this sentence a bit weird? "Just as it did in when developing locally..."

Reviewer 2: The 'in' may be superfluous; `Just as it did when developing locally...`

Author: Totally right, great catch, will edit!
